gigasecond/test_gigasecond.py

```diff
@@ -0,0 +1,21 @@
+from gigasecond import gigasecond
+from datetime import datetime, timedelta
+
+seconds = timedelta(seconds=1000000000)
+
+
+def test_recently_apast_date_time():
+    assert gigasecond(datetime(2011, 4, 25)) == datetime(2011, 4, 25) + seconds
+
+
+def test_very_old_date_time():
+    assert gigasecond(datetime(1886, 2, 11)) == datetime(1886, 2, 11) + seconds
+
+
+# def test_todays_date():
+#     # import pdb; pdb.set_trace()
+#     assert gigasecond(datetime.now()) == datetime.now() + seconds
+
+
+def test_future_date_time():
+    assert gigasecond(datetime(2020, 4, 11)) == datetime(2020, 4, 11) + seconds
```

Comment:
You can extract `datetime(2020, 4, 11)` as a named variable, otherwise it's duplication.